benchmarks/lambda/config.yml

```diff
@@ -25,4 +25,4 @@ lambda:results:
   stage: collect
   needs: ["lambda:simulate"]
   script:
-    - python benchmarks/lambda/analysis/lambda_plots.py results/lambda/results_epic_zdc_sipm_on_tile_only_lambda_dec
+    - snakemake --cores 1 results/epic_zdc_sipm_on_tile_only/lambda
```

Comment:
Looking at what's currently running in the checks, it appears that the collect stage is just rerunning the previous stage; ie, the sim_output is not being persisted from one stage to the other.

Reply:
Yes, we need a step that looks more like a benchmark.